src/dbus_mpris.rs

```diff
@@ -610,6 +637,88 @@ fn create_dbus_server(
     let async_connection = AConnection::new(connection.clone(), handle)
         .expect("Failed to create async dbus connection");
 
+    let connection2 = connection.clone();
+
+    let f2 = rx.for_each(move |message| {
+        match message {
+            TrackChange::Changed(changed_track) => {
+                println!("TrackChange::Changed");
+
+                let track_name2: Variant<Box<dyn RefArg>> =
+                    Variant(Box::new(changed_track.track.to_string()));
+
+                let album: Variant<Box<dyn RefArg>> =
+                    Variant(Box::new(changed_track.album.to_string()));
+                let artist: Variant<Box<dyn RefArg>> = Variant(Box::new(changed_track.artist));
```

Comment:
For the GNOME media control, this only works for the initial song playing for me, for it to work on track changes, I modified it to:

```
let artist: Variant<Box<dyn RefArg>> = Variant(Box::new(vec![changed_track.artist.to_string()]));
```

It looks like in the artist should be a [list of strings](https://www.freedesktop.org/wiki/Specifications/mpris-spec/metadata/#index6h4).

(I don't know rust, but really want this feature!).